.github/workflows/custom-actions/build-packager/action.yaml

```diff
@@ -0,0 +1,109 @@
+name: Build Shaka Packager
+
+description: |
+  A reusable action to build Shaka Packager.
+  Leaves build artifacts in the "artifacts" folder.
+
+inputs:
+  os_name:
+    description: The name of the OS (one word).  Appended to artifact filenames.
+    required: true
+  lib_type:
+    description: A library type, either "static" or "shared".
+    required: true
+  build_type:
+    description: A build type, either "Debug" or "Release".
+    required: true
+  build_type_suffix:
+    description: A suffix to append to the build type in the output path.
+    required: false
+    default: ""
+  exe_ext:
+    description: The extension on executable files.
+    required: false
+    default: ""
+
+runs:
+  using: composite
```

Comment:
Just to confirm, for composite runs, the next step will not run if the current step fails, right?

Reply:
Correct.